packages/grafana-ui/src/components/ThresholdsEditor/_ThresholdsEditor.scss

```diff
@@ -86,7 +86,6 @@
 
 .thresholds-row-input-inner-color-colorpicker {
   border-radius: 10px;
-  overflow: hidden;
```

Comment:
This is not needed as the default trigger is properly rounded. `border-radius: 10px` though needs to stay otherwise the box-shadow defined here would be not be round.